public/app/features/explore/Logs/LogsTable.tsx

```diff
@@ -45,6 +47,11 @@ export function LogsTable(props: Props) {
   const [tableFrame, setTableFrame] = useState<DataFrame | undefined>(undefined);
   const timeIndex = logsFrame?.timeField.index;
 
+  const { currentData: logsDrilldownData, error: logsDrilldownError } =
+    logsdrilldownAPI.endpoints.listLogsDrilldown.useQuery({});
+
+  const [createLogsDrilldown] = logsdrilldownAPI.endpoints.createLogsDrilldown.useMutation();
```

Comment:
Instead of doing this, you can import the hooks directly and call them like so: 
```ts
  const { currentData: logsDrilldownData, error: logsDrilldownError } =
    useListLogsDrilldownQuery({})

  const [createLogsDrilldown] = useCreateLogsDrilldownMutation();
```